src/Core/ServiceWrapper/WrapperService.cs

```diff
@@ -285,6 +285,7 @@ protected override void OnStart(string[] args)
             Log.Info("Starting " + this.descriptor.Executable + ' ' + startArguments);
 
             // Load and start extensions
+            Console.WriteLine("Loading extensinos");
```

Comment:
Remember to remove all `Console.WriteLine` when you finish this.

Reply:
Thanks for reminding!

Reply:
Reminder :)